README.md

```diff
@@ -6,7 +6,7 @@ This Gem provides a Ruby interface to [the Pusher HTTP API for Pusher Channels](
 
 ## Supported Platforms
 
-* Ruby - supports **Ruby 2.4 or greater**.
+* Ruby - supports **Ruby 2.7 or greater**.
```

Comment:
```suggestion
* Ruby - supports **Ruby 2.6 or greater**.
```

Looks like the 2.6 test passes. We can add a required ruby version with the minimum ruby version supported.
https://guides.rubygems.org/specification-reference/#required_ruby_version=